### PR TITLE
arcade-machine now scrolls whenever an element is focused

### PR DESCRIPTION
**Pull request by likewei92**
It used to scroll only upon directional changes, now it scrolls when focus is manually set as well

### CODE REVIEW

File: src/focus.service.ts [hunk 1]
```diff
@@ -275,15 +275,19 @@ export class FocusService {
     this.registrySubscription = this.registry
       .setFocus
       .filter((el: HTMLElement) => !!el)
-      .subscribe((el: HTMLElement) => this.selectNode(el));
+      .subscribe((el: HTMLElement) => {
+        this.rescroll(el, scrollSpeed, this.root);
```
Comment (connor4312) [inline src/focus.service.ts]: Redundant

Comment (likewei92) [inline src/focus.service.ts]: Why are these redundant?

Comment (likewei92) [inline src/focus.service.ts]: I just tested and I don't think either of those are redundant?

Comment (connor4312) [inline src/focus.service.ts]: My bad. Instead these should be put to selectNode.

**Pull request merged**